pages/platform/ai/training_tuto_07_train_rasa_chatbot/guide.en-gb.md

```diff
@@ -0,0 +1,210 @@
+
+### Test your model
+
+So, you've trained your model and you want to see if he is returning good results. Let's go ! To do this, let's create a VScode notebook. You can run this command to be fast :
+```bash
+ovhai notebook run conda vscode \
+	--name vscode-ovh-chatbot \
+	--framework-version conda-py39-cuda11.2-v22-4 \
+	--volume <model-output-container>@GRA:/workspace/trained-models:RW \
+	--cpu 10 \
+	--token <token> \
+```
+Then install all the requirements by running in a terminal : 
+```bash 
+pip install -r ~/ai-training-examples/jobs/rasa-chatbot/requirements_rasa.txt 
+```
+
+Now we can test our model !
+
+Open two terminals. In one of the terminal we will run one custom action and in the second, we will speak with the chatbot. Here is the command for the first terminal : 
+
+```bash
+cd ~/ai-training-examples/jobs/rasa-chatbot
+rasa run actions 
+```
+
+In rasa, an interaction with a user is described by an intent from an user and a response from the chatbot. The response from the chatbot is called an action. Sometimes and in this case, we need to create some custom actions (filling a form with the user and collect some data). This is the purpose of the command `rasa run actions`. 
+
+In the other terminal, launch: 
+
+```bash
+rasa shell -m ~/trained-models --endpoints ~/ai-training-examples/jobs/rasa-chatbot/endpoints.yml
+```
+
+Explication of the command : 
+- `rasa shell` : load a model and speak with him to test it
+- `-m ~/trained-models` : This is to specify the path of where your model is saved. As see before, your model has been trained and is save in this folder. If you have several models in the folder `trained-models`, you can specify the specific name of the model you want to train in this argument. 
+- `--endpoints ~/ai-training-examples/notebooks/natural-language-processing/chatbot/conda/rasa_bot/endpoints.yml` : You specify the way of your endpoints.yml file. This file is used to tell where your model is running and the where the port of your custom actions are running. 
+
+This is a small example of what you can get with this command : 
+
+![image](images/discussion.png){.thumbnail}
+
+
+Congrats ! you can test your first rasa model and speak with him. If you have several models in your folder trained-models, you can specify after the option -m the folder and the specific model you want to train. 
+
+If you're not satisfied about the model because your chatbot doesn't respond very well, you can run this command. It will run the job again and create a new model.
+
+```bash
+ovhai job rerun <job_id> 
+```
+
+To get the job id of the previous job, just run this command to get the list of the job you've run before. 
+
+```bash
+ovhai job ls
+```
+
+More explanation are here : [CLI Reference](https://docs.ovh.com/gb/en/publiccloud/ai/cli/overview-cli/).
+
+
+  Once you have your model is ready, we must deploy the model to use it. This will be ensure with the tool AI Deploy from the public cloud.
```

Comment:
```suggestion
Once you have your model is ready, we must deploy the model to use it. This will be ensure with the tool AI Deploy from the Public Cloud.
```